src/api/todos.ts

```diff
@@ -0,0 +1,20 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const USER_ID = 1106;
```

Comment:
tip (ignore this for this project, just a reminder): always use .env for these cases of api keys